ibm_cloud_sdk_core/base_service.py

```diff
@@ -65,7 +65,7 @@ def __init__(self,
 
         if display_name:
             service_name = display_name.replace(' ', '_').lower()
```

Comment:
I think we also need to replace '-' with '_' in the service_name value.    Some values for the vcap services name have -'s in them.